modules/unsupported/arcgis-rest/pom.xml

```diff
@@ -134,8 +131,8 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.8</source>
-                            <target>1.8</target>
+                            <source>11</source>
```

Comment:
We still build targetting java 8, that's the minimum version, the goal is just to have it buliding and running on both versions.
That said, this is just a tool that needs to be called on demand? It's not a problem if it works only with java 11 I guess.

Reply:
I made changes to the POMs to make them work with both JDKs.